index.html

```diff
@@ -120,5 +120,32 @@ <h2 class="en">About me</h2>
         >View Project</a
       >
     </section>
+    <section class="form">
+      <form>
+        <h2 class="custom-font form-legend"> 
+          I'm always interested in hearing about new projects, so if you'd like to
+          chat please get in touch.
+        </h2>
+        <ul class="d-flex-colum">
+          <li>
+            <label for="name"></label>
+            <input class="input-text-email custom-font" type="text" id="name" placeholder="Full Name">
+          </li>
+          <li>
+            <label for="mail"></label>
+            <input class="input-text-email custom-font" type="email" id="mail" placeholder="Email Adress" required>
+          </li>
+          <li class="text-area-container">
+            <label for="msg"></label>
+            <textarea class="textarea-message custom-font" name="message" id="msg" cols="30" rows="10" placeholder="Enter Text Here">
+            </textarea>
+            <img class="resizer-handle" src="./pictures/text_area/resizer_handle.png" alt="">
+          </li>
+          <li>
+            <button class="last-b custom-font" type="submit">Get in touch</button>
```

Comment:
The user cannot submit the form if one or more of the fields is empty, now I can submit the form with only the email
Kindly fix that